packages/react-components/src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -298,6 +343,11 @@ export const RichTextEditor = React.forwardRef<RichTextEditorComponentRef, RichT
   );
 
   useEffect(() => {
+    /* @conditional-compile-remove(rich-text-editor-image-upload) */
```

Comment:
Whey do we get and then set with same content here?

Reply:
We get the prevInlineImage from the initialContent and set into the state